benches/boxed_uint.rs

```diff
@@ -11,7 +11,7 @@ fn bench_shifts(c: &mut Criterion) {
     group.bench_function("shl_vartime", |b| {
         b.iter_batched(
             || BoxedUint::random(&mut OsRng, UINT_BITS),
-            |x| black_box(x.shl_vartime(UINT_BITS / 2 + 10)),
+            |x| black_box(x.overflowing_shl(UINT_BITS / 2 + 10).0),
```

Comment:
Why the change here? And if you're benchmarking `overflowing_shl` now, the string label should be changed too.

Reply:
Yeah sorry I think I got really confused between the various `shl`/`shr` functions.